addons/mrp/migrations/10.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,424 @@
+---Fields in module 'mrp'---
+
+# Good reference for tracking changes between versions:
+# https://www.scribd.com/document/343980661/oDoo-10-Manufacturing
+# https://github.com/odoo/odoo/commit/2ddc35a5300e89981c0572ae6749aee946f34e9d
+
+mrp          / ir.attachment            / priority (selection)          : NEW selection_keys: ['0', '1', '2', '3']
+# NOTHING TO DO
+
+mrp          / mrp.bom                  / date_start (date)             : DEL
+mrp          / mrp.bom                  / date_stop (date)              : DEL
```

Comment:
At the very least BoMs that are not valid anymore should be archived for v10, otherwise they'll suddenly be possibly used again during production as a valid BoM. Not sure what could be done with date_start in future BoMs. Also date_stop values in the future will have information lost, but alas, I guess nothing that can be done about that, other than warn in some documentation or whatnot.